tmt/convert.py

```diff
@@ -154,12 +155,15 @@ def write_markdown(path, content):
         raise ConvertError(f"Unable to write '{path}'.")
 
 
-def add_bug(bug, data, system=SYSTEM_BUGZILLA):
-    """ Add relevant bug into data under the 'link' key """
+def add_link(link, data, system=SYSTEM_BUGZILLA, type_='relates'):
```

Comment:
you mean as in `def add_target` or target argument instead of link?
as this is working with link attributes then changing function name doesn't seem right, however the argument might be better as target